src/main/thrift/parquet.thrift

```diff
@@ -576,5 +631,8 @@ struct FileMetaData {
    * e.g. impala version 1.0 (build 6cf94d29b2b7115df4de2c06e2ab4326d721eb55)
    **/
   6: optional string created_by
+
+  /** Sort order used for each column in this file */
+  7: optional list<ColumnOrder> column_orders;
```

Comment:
How do you know from 1 ColumnOrder which column it applies to? It's position in this list has to match the position in the `schema` list above? If that's the case I think we should add a comment for that here. I guess this has to be here to avoid repeating it in each Statistics object?